codespell_lib/data/dictionary.txt

```diff
@@ -12939,8 +12939,11 @@ sheduled->scheduled
 shedules->schedules
 sheduling->scheduling
 sheild->shield
+sheme->scheme
+shepard->shepherd
```

Comment:
https://en.wikipedia.org/wiki/Shepard

Reply:
removed

Reply:
it's still part of this PR

Reply:
You are right.... uhm... maybe something wrong in the rebase?

Now I fix it and then check.